tests/framework-tests/core/js/DataBindingTests.js

```diff
@@ -2242,7 +2242,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 options: {
                     moduleSource: {
                         funcName: "fluid.tests.fluid5659source",
-                        args: "{fluid5659relay}.options.lang"
+                        args: ["{fluid5659relay}.options.lang"]
```

Comment:
It is actually the reverse - it used to be that args being an array was the only possibility - and hence binding it to an array value would represent the whole of args. This was really a dispatching bug. Now it is possible that args is a single argument, and hence to remove the ambiguity we need to explicitly represent the whole argument array - that is, "lang" is an array, and the function fluid5659source expects a single argument which is an array.